backend/de.metas.acct.base/src/test/java/de/metas/acct/model/validator/C_BPartnerTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * #%L
+ * de.metas.acct.base
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.acct.model.validator;
+
+import de.metas.acct.AcctSchemaTestHelper;
+import de.metas.acct.api.AcctSchemaId;
+import groovy.lang.Tuple3;
```

Comment:
please avoid using Tuple3 - we don'T want additional dependencies to groovy, weven if the jar is still among the dependencies.
Feel free to create a similar local class hat is annotated with lombok `@Value¸

Reply:
created BpartnerValueToDebtorCreditorIds.